From 89677f815f685e06b937bbe02d4d75d7f09e16dd Mon Sep 17 00:00:00 2001
From: Nox W <wewilair@microsoft.com>
Date: Sat, 10 Dec 2022 17:09:03 +0000
Subject: [PATCH 2/4] Add root CMakeLists.txt

---
 CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 CMakeLists.txt

diff --git a/CMakeLists.txt b/CMakeLists.txt
new file mode 100644
index 0000000..d364daf
--- /dev/null
+++ b/CMakeLists.txt
@@ -0,0 +1,5 @@
+cmake_minimum_required (VERSION 3.1.0)
+
+add_subdirectory(src)
+
+
-- 
2.25.1

