From e765e7afdd9b169444ccdc2a781cf2ec9ee332e5 Mon Sep 17 00:00:00 2001
From: Nox W <wewilair@microsoft.com>
Date: Sat, 10 Dec 2022 17:21:32 +0000
Subject: [PATCH 3/4] Fix io_utility CMakeLists.txt

---
 src/io_utility/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/io_utility/CMakeLists.txt b/src/io_utility/CMakeLists.txt
index 62349af..bd090c2 100644
--- a/src/io_utility/CMakeLists.txt
+++ b/src/io_utility/CMakeLists.txt
@@ -60,7 +60,7 @@ if (BSDIFF_INCLUDE_DIR STREQUAL "BSDIFF_INCLUDE_DIR-NOTFOUND")
 endif()
 
 # NOTE: Comment our for ADU Yocto
-#find_package(ZLIB REQUIRED)
+find_package(ZLIB REQUIRED)
 #find_package(zstd CONFIG REQUIRED)
 #find_package(Microsoft.GSL CONFIG REQUIRED)
 #target_link_libraries(io_utility PRIVATE Microsoft.GSL::GSL)
-- 
2.25.1

