From 16f4c073208f3a9029c3d3c4b774f7fe0f9bd5bc Mon Sep 17 00:00:00 2001
From: Nox W <wewilair@microsoft.com>
Date: Thu, 15 Dec 2022 00:01:43 +0000
Subject: [PATCH 4/4] Add missing header in zstd_decompression_reader

---
 src/io_utility/zstd_decompression_reader.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/io_utility/zstd_decompression_reader.h b/src/io_utility/zstd_decompression_reader.h
index 5ccc9c3..e669ec2 100644
--- a/src/io_utility/zstd_decompression_reader.h
+++ b/src/io_utility/zstd_decompression_reader.h
@@ -8,6 +8,7 @@
 
 #include <vector>
 #include <map>
+#include <limits>
 
 #include "zstd_wrappers.h"
 #include "zstd.h"
-- 
2.25.1

